.github/workflows/docker-release.yml

```diff
@@ -0,0 +1,50 @@
+name: Docker-Release
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    if: "!contains(github.event.head_commit.message, '[ci skip]')"
+
+    steps:
+
+    - name: Set up Go 1.13
+      uses: actions/setup-go@v1
+      with:
+        go-version: 1.13
+      id: go
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v1
+
+    - name: Set version output
+      id: vars
+      run: echo ::set-output name=tag::${GITHUB_REF:11} # tag name without leading 'v'
```

Comment:
The reason I dropped `v` from Docker tag was because anything after `:` is considered some sort of a _version_ and notion of `v` in that was kind of redundant. There are so many cases of both sides out there, two very notable ones:

- Grafana [tags](https://github.com/grafana/grafana/tags) with leading `v` BUT [images](https://hub.docker.com/r/grafana/grafana/tags) without.
- Prometheus [tags](https://github.com/prometheus/prometheus/tags) with leading `v` AND [images](https://hub.docker.com/r/prom/prometheus/tags) also with.

At the end, this was a personal preference. And to be clear I'm not affiliated with Tailscale and this PR is only a personal contribution.